imap_processing/codice/codice_l1a.py

```diff
@@ -8,11 +8,9 @@
     from imap_processing.codice.codice_l0 import decom_packets
     from imap_processing.codice.codice_l1a import codice_l1a
     packets = decom_packets(packet_file)
-    cdf_filename = codice_l1a(packets)
+    dataset, cdf_filename = process_codice_l1a(packets)
```

Comment:
No need to change in this PR, I'm just throwing this idea out, but what about adding the filename to the dataset as an attribute? `dataset.cdf_filename = cdf_filename` or `dataset.attrs["filename"] = cdf_filename`
so that later you could just grab it from the dataset without needing to pass tuples around.

Reply:
I like it!